ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -94,22 +99,20 @@ module('Acceptance | sync | overview', function (hooks) {
         return {};
       });
 
-      assert.dom('[data-test-badge-namespace]').hasText('foo'); // confirm we're in admin/foo
+      // confirm we're in admin/foo
+      assert.dom('[data-test-badge-namespace]').hasText('foo');
+
       await click(ts.navLink('Secrets Sync'));
-      assert.dom(ts.overview.optInBanner).exists('Opt-in banner is shown');
```

Comment:
didn't think we needed these extra dom assertions since the integration test covers all of those things being present. this way the two tests in this module are more specifically focused on making sure the requests are made to the right namespace (and not worried about the banner/modal)